package.json

```diff
@@ -141,7 +145,7 @@
     "request-promise": "^4.2.2",
     "request-promise-native": "^1.0.5",
     "rimraf": "^3.0.2",
-    "sequelize": "^6.6.2",
```

Comment:
_Suggestion_: Version updates not part of this PR (or as separate commit, which can be merged as a single change)

Reply:
Ah, good catch, I wanted to drop that commit again, but forgot. I only did this because the build failed at the `npm audit` stage because of the the ansi-regex vulnerability and didn't even get to the commitlint step on CircleCI. I'll definitely remove that again.

Reply:
I have dropped that commit.